lib/streamlit/web/server/server.py

```diff
@@ -95,6 +105,10 @@
     r"(?:script-health-check|_stcore/script-health-check)"
 )
 
+OAUTH2_CALLBACK_ENDPOINT: Final = "/oauth2callback"
```

Comment:
@kajarenc should be cool to be configurable with toml

Reply:
At this stage, we don't plan to make it configurable, since this required during Tornado app initialization.